src/cljs/main/broadfcui/page/method_repo/method/details.cljs

```diff
@@ -76,7 +76,10 @@
              :href (if (common/has-terra-return?)
                      (let [{:keys [namespace name]} (:dest-config-id @state)]
                        (string/join "/" [(config/terra-url) (nav/get-link :workspace-summary (:dest-workspace-id @state)) "tools" namespace name]))
-                     (nav/get-link :workspace-method-config (:dest-workspace-id @state) (:dest-config-id @state)))}}])
+                     (if (common/has-firecloud-return?)
+                       (let [{:keys [namespace name]} (:dest-config-id @state)]
+                         (string/join "/" [(config/firecloud-terra-url) (nav/get-link :workspace-summary (:dest-workspace-id @state)) "tools" namespace name]))
+                       (nav/get-link :workspace-method-config (:dest-workspace-id @state) (:dest-config-id @state))))}}])
```

Comment:
this has the `if (common/has-firecloud-return?)` nested *inside* the  `(if (common/has-terra-return?)` at line 76 - I don't think that's correct.

Reply:
I was trying to do an `if-elif-else` type thing here, but I think `cond` would work nicely here.